### DISCUSSION TITLE
can't get student developer pack

### DISCUSSION THREAD

**Discussion by atreyeekar** [community/community#55434] [GitHub Education]
### How resolve this issue? Please help me.
![Screenshot 2023-05-14 143801](https://github.com/community/community/assets/133365663/4fe34192-0e10-4d92-94ff-86000c6c4319)

**Comment by Ntshangase** [comment]
Hello @atreyeekar 

- Nothing to worry about, this is perfectly normal, I would suggest that you try applying again in a few days, as your account is still relatively new
- In the meantime you can create a new repository to have some activity in your account.

**Comment by katikuuu** [comment]
Hey @atreyeekar
Try using the calling letter to the school. It is easier to verify your application With that than the School ID

**Comment by Shariar-Hasan** [comment]
Dont worry about it, do what the instruction says.
just apply for the student pack, after a few days.

And also make sure to have a clear and unblurred picture of the document you provided.

i hope you will get your student pack, within a short time after applying.
Hope you got your answer.
